test/integration/generated/bodyComplex/src/models/parameters.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+import * as coreHttp from "@azure/core-http";
+
+export const $host: coreHttp.OperationParameter = {
```

Comment:
It is not a change in the modeler but on how we're now handling the parameters. In master, we're inlining the parameter definition for requestBody in the operation. Here, I'm defining them in the parameters file to be referenced instead of duplicating them in the operation Specs.

i.e. https://github.com/Azure/autorest.typescript/blob/12509e48b39420721dfc344a816c10465260e2b3/test/metadata/generated/BodyComplex/src/operations/arrayModel.ts#L168

and 

https://github.com/Azure/autorest.typescript/blob/12509e48b39420721dfc344a816c10465260e2b3/test/metadata/generated/BodyComplex/src/operations/arrayModel.ts#L206

However looking at the generated parameters closely, I think I need to make a few changes, for instance, I need to reference Mappers for composite ones that exist in Mappers.ts, also need to figure out why some parameterPaths are missing. Thanks for bringing this up 👍